server/grpc_service.go

```diff
@@ -712,10 +732,6 @@ func (s *Server) ScatterRegion(ctx context.Context, request *pdpb.ScatterRegionR
 		region = core.NewRegionInfo(request.GetRegion(), request.GetLeader())
 	}
 
-	if rc.IsRegionHot(region) {
```

Comment:
Please explain the reason for canceling this judgment.

Reply:
https://github.com/tikv/pd/blob/3c625632b06c956ab74ca6e6a2941f6888200071/server/schedule/region_scatterer.go#L270-L287

duplicated with these checks.